applications/app/views/offlinePage.scala.html

```diff
@@ -14,10 +17,65 @@
     @* get the stylesheets downloading ASAP *@
     @fragments.stylesheets(None)
 
-    @* load the app *@
+    @* We don't use the inlineJSBlocking template because we need to specify
+    our own paths *@
```

Comment:
I still think dryness only makes sense when duplication is excessive, and it allows the reader to better understand the code. I think `alwaysUseBundlesInDev` makes little sense to others, because it's not clear why or when it's needed. Adding a parameter in our main config system for the purpose of a single page doesn't feel right.

Ideally, I don't want to think about the offline page system when I'm doing online page work (~100% of my work). Yes, this implies I might break the offline page, in my ignorance. I'm happy with that, because I never agreed to maintaining it, nor should the whole team have that burden. Which is why I believe a stand-alone, isolated config is preferable.